README.md

```diff
@@ -62,7 +62,7 @@ To learn more about the different rule types, their guarantees and how they are
 #### Carrying out software supply chain steps
 
 ##### in-toto-run
-`in-toto-run` executes the passed command and records the path and hash of the passed *materials* - files before command execution - and *products* - files after command execution and optionally stores them together with the command's *byproducts* (e.g: return value, stdout or stderr) to a link file (`<step-name>.link`), signed with the functionary's key.
+`in-toto-run` generates link metadata for the given command-line option. See the [this simple usage example from the demo application for more details](https://github.com/in-toto/demo#tampering-with-the-software-supply-chain).
```

Comment:
I don't know how clearer this new message is. I think it's worthy to mention that the command-line option passed is a *command* that will be run (that's why the previous phrasing was a little odd...)